cmd/cosign/cli/attest/attest.go

```diff
@@ -214,10 +214,9 @@ func (c *AttestCommand) Exec(ctx context.Context, imageRef string) error {
 		return err
 	}
 
-	se, err := ociremote.SignedEntity(digest, ociremoteOpts...)
-	if err != nil {
-		return err
-	}
+	// We don't actually need to access the remote entity to attach things to it
+	// so we use a placeholder here.
+	se := ociremote.SignedUnknown(digest)
```

Comment:
I worry that Digest isn't clear enough, since it is effectively what `SignedEntity` is (especially after my PR to add `Digest` to `oci.SignedEntity`).

That said, I don't feel particularly strongly about the naming here `:bikeshed:`